include/unifex/type_erased_stream.hpp

```diff
@@ -107,18 +126,23 @@ struct _stream<Values...>::type {
       continuation_info get_continuation_info() const noexcept override {
         return continuation_info::from_continuation(receiver_);
       }
+
+      any_scheduler_ref get_scheduler() const noexcept override {
+        return scheduler_;
```

Comment:
Do you need the scheduler to be a member variable? Seems that you could delay evaluation of `get_scheduler` until this point and save on some byes.

Reply:
get_scheduler returns schedulers by value in all of the other cases, so this class needs to store the downstream receivers scheduler by value as a member to ensure that the scheduler reference passed to any_scheduler_ref lives beyond the scope of this function call.

Reply:
Although not documented as a requirement: all existing `get_scheduler()` implementations return by value. If we returned inline, `any_scheduler_ref` would take a temporary.